packages/phishing-controller/CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.0.0]
+
+### Uncategorized
+
+- Feat: add allowlist to c2 domain detection ([#4464](https://github.com/MetaMask/core/pull/4644))
+- Feat: Block Client Side C2 Requests by Managing a hashed C2 Request Blocklist ([#4526](https://github.com/MetaMask/core/pull/4526))
```

Comment:
The changelog entries in the PR descriptions for these will need to be moved here and distributed to the appropriate categories. 

Also entries should be added for the dependency additions: https://github.com/MetaMask/core/pull/4526#discussion_r1735077493